docs/setup/webhooks.md

```diff
@@ -131,7 +131,7 @@ if (data.counter === undefined) {
 
 ### V1 API
 
-The v1 API expects `result` to be a string. The string will be automatically transformed into markdown. All webhook messages
+The v1 API expects `result` to be a string. The string will be automatically interpreted as markdown and transformed into HTML. All webhook messages
```

Comment:
```suggestion
The v1 API expects `result` to be a string. The string will be automatically interpreted as Markdown and transformed into HTML. All webhook messages
```

Reply:
I agree this appears to be the right spelling for prose. I checked the rest of the codebase for the same and there appear to be none, except the comments in the code don't capitalize at all